### PR TITLE
[UXE-2111] feat: inserting properties in the edge application failed event

### PR DESCRIPTION
**Pull request by pauloSF0**
## Pull Request
[UXE-2111]

### What is the new behavior introduced by this PR?
Inserting properties in the edge application failed event

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Captura de Tela 2024-03-15 às 15 50 15

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-2111]: https://aziontech.atlassian.net/browse/UXE-2111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/services/edge-application-services/edit-edge-application-service.js [hunk 1]
```diff
@@ -73,17 +73,21 @@ const adapt = (payload) => {
  * @param {Object} httpResponse.body - The response body.
  * @returns {string} The result message based on the status code.
  */
-const extractApiError = (httpResponse) => {
-  const errorKeys = Object.keys(httpResponse.body)
-  const noProductErrorFound = errorKeys.includes('user_has_no_product')
-
-  if (noProductErrorFound) {
-    const noProductErrorMessage =
-      'In order to perform this action, you must first have access to the product Tiered Cache'
```
Comment (HerbertJulio) [inline src/services/edge-application-services/edit-edge-application-service.js:82]: Paulo essa mensagem realmente vai ser removida?

Comment (pauloSF0) [reply src/services/edge-application-services/edit-edge-application-service.js:82]: Eu vi essa mensagem, mas não estamos só mostrando o que está vindo da API ?

Comment (pauloSF0) [reply src/services/edge-application-services/edit-edge-application-service.js:82]: Ou nessa ai tem que ter essa analise

Comment (HerbertJulio) [reply src/services/edge-application-services/edit-edge-application-service.js:82]: Recomendo verifica com o @vinigfer , de qualquer forma recomendo não subi isso neste PR.

**Review by lucasmendes21 — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

**Pull request merged**